### PR TITLE
Merge 14.x to main.

### PR DESCRIPTION
**Pull request by davidlehn**
The branches and release style in this repo got a bit awkward.  This commit is merging the active v14.x branch back on to main.  The latest code and primary releases will then continue to be on main, and version branches (v13.x, v14.x, v15.x, etc) will branch off of main as needed.  Handling releases and network versions will be done per version as needed and with documentation.

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
Never thought I'd live to see the day!! :) (that this branch would merge to main)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,100 @@
...
+## 14.0.0 -
+
+### `14.0.0-beta.2` - 2021-08-19
+
+### Added
+- Add `driver.getInitial()` method, to explicitly construct a DID Document
+  deterministically, from a cryptonym DID.
+- Add an optional `seed` param to `generate()` to generate DID document from
+  a 32-byte array seed.
+
```
Comment (dlongley) [inline CHANGELOG.md:12]:
```suggestion
```

**Review by dlongley — APPROVED** [review]
Look large to review other than a skim ... approving given that this looks like the right path forward per our earlier discussion.

File: CHANGELOG.md (hunk 1 above)
Comment (davidlehn) [reply CHANGELOG.md:12]: This is in the 14.x branch.  The module conversion branch, aka 15.x, cleans up the changelog.  Has a fix for this and adds entries for other 14.x releases.  I think we can leave this blank line atrocity here for now.

**Comment by davidlehn** [comment]
Regular GitHub merge and rebase did not want to cooperate.  It tried to create another merge instead of just doing what was in this pr.  Force fixed on command line to minimize non-linear history and clean up the versioning and branch situation.

**Pull request merged**